okgo/checker.go

```diff
@@ -27,13 +27,18 @@ import (
 
 type CheckerPriority int
 
+type CheckerMultiCPU bool
+
 type Checker interface {
 	// Type returns the type of this Checker.
 	Type() (CheckerType, error)
 
 	// Priority returns the priority of the check. A lower number indicates a higher priority (will be run earlier).
 	Priority() (CheckerPriority, error)
 
+	// MultiCPU returns if the check uses multiple CPUs  in its check command
```

Comment:
Remove extra space in comment

```suggestion
	// MultiCPU returns if the check uses multiple CPUs in its check command
```

Reply:
Done